src/main/kotlin/com/openlattice/shuttle/pods/ShuttleMapstoresPod.kt

```diff
@@ -0,0 +1,27 @@
+package com.openlattice.shuttle.pods
+
+import com.kryptnostic.rhizome.mapstores.SelfRegisteringMapStore
+import com.openlattice.hazelcast.mapstores.shuttle.IntegrationJobsMapstore
+import com.openlattice.hazelcast.mapstores.shuttle.IntegrationsMapstore
+import com.openlattice.postgres.PostgresPod
+import com.openlattice.shuttle.Integration
+import com.openlattice.shuttle.IntegrationJob
+import com.zaxxer.hikari.HikariDataSource
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.context.annotation.Import
+import java.util.*
+import javax.inject.Inject
+
+@Configuration
+@Import(PostgresPod::class)
```

Comment:
This should probably change to `MissionParametersPod::class` if removing `postgres.yaml` 😩
Probably simpler to just keep postgres config in `postgres.yaml`